modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -320,68 +382,118 @@ It cannot be set when an individual replication is created, or when its settings
 The default is 0, meaning no limit is applied.
 For information, see xref:xdcr-reference:xdcr-advanced-settings.adoc#table-of-xdcr-advanced-settings[XDCR Advanced Settings].
 
+This setting can be established and retrieved either for an individual replication or globally.
+
 | `optimisticReplicationThreshold`
 | Integer (0 to (20*1024*1024))
 | Default: 256.
 Documents with sizes less than this threshold (in bytes) will be replicated optimistically.
 
+This setting can be established and retrieved either for an individual replication or globally.
+
 | `pauseRequested`
 | Boolean (true or false)
 | Default: false.
 Indicates whether the replication has been issued a pause request.
+
 This setting can only be established for and retrieved from an individual replication: it cannot be established or retrieved as part of global settings.
 
 | `priority`
-| `High`, `Medium`, or `Low`
-| Default: `High`.
+| High, Medium, or Low
+| Default: High.
 Resource-allocation for the replication.
 For information, see xref:learn:clusters-and-availability/xdcr-overview.adoc#xdcr-priority[XDCR Priority].
 
+This setting can be established and retrieved either for an individual replication or globally.
+
+| `retryOnRemoteAuthErr`
+| Boolean (true or false)
+| Whether to retry connection for a replication that has failed due to a problem in authenticating with the target cluster.
+The default is true.
+
+This setting can be established and retrieved either for an individual replication or globally.
+
+| `retryOnRemoteAuthErrMaxWaitSec`
+| Integer
+| The maximum number of seconds to wait before retrying a connection that failed due to a problem in authenticating with the target cluster.
+This only takes effect when `retryOnRemoteAuthErr` is true.
+
+This setting can be established and retrieved either for an individual replication or globally.
+
 | `sourceNozzlePerNode`
 | Integer (1 to 100)
 | The number of nozzles that can be used for this replication per source cluster node.
 This together with `target_nozzle_per_node` controls the parallelism of the replication.
 
+This setting can be established and retrieved either for an individual replication or globally.
+
 | `statsInterval`
 | Integer (200 to 600000)
 | Default: 1000.
 The interval (in milliseconds) for statistics updates.
 
+This setting can be established and retrieved either for an individual replication or globally.
+
 | `targetNozzlePerNode`
 | Integer (1 to 100)
 | The number of outgoing nozzles per target node.
 This together with `source_nozzle_per_node` controls the parallelism of the replication.
 
+This setting can be established and retrieved either for an individual replication or globally.
+
 | `type`
-| String (`capi` or `xmem`)
-| Default: `xmem`.
-Note that `capi` and `xmem`, are respectively represented by *Version 1* and *Version 2*, in Couchbase Web Console.
+| String ("capi" or "xmem")
+| Default: "xmem".
+The replication type, which can be "capi" or "xmem".
```

Comment:
CAPI is now deprecated - so xmem is the only value

Reply:
I have changed to specify xmem only.